StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodMessagingElement/PMME+API.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  PMME+API.swift
+//  StripePaymentSheet
+//
+//  Created by George Birch on 10/9/25.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+
+private let pmmeApiEndpoint = URL(string: "https://ppm.stripe.com/config")!
+
+extension PaymentMethodMessagingElement {
+    static func get(configuration: Configuration) async throws -> APIResponse {
```

Comment:
Can we add a quick docstring for this method explaining what it returns, the method name`PaymentMethodMessagingElement.get` is not immediately obvious